src/video_core/renderer_vulkan/vk_platform.cpp

```diff
@@ -86,6 +86,10 @@ vk::SurfaceKHR CreateSurface(vk::Instance instance, const Frontend::WindowSDL& e
             UNREACHABLE();
         }
     } else if (window_info.type == Frontend::WindowSystemType::Wayland) {
+        if (Config::isRdocEnabled()) {
+            LOG_ERROR(Render_Vulkan, "As of today RenderDoc and Wayland are incompatible!");
```

Comment:
```suggestion
            LOG_ERROR(Render_Vulkan, "RenderDoc is not compatible with Wayland, use an X11 window instead.");
```

nit: `today` is a bit vague and easily outdated, and can suggest the alternative here.

Reply:
Nit of the nit, I assume "not incompatible" should probably be "not compatible"

Reply:
> Nit of the nit, I assume "not incompatible" should probably be "not compatible"

Fixed the suggestion